backend/globaleaks/handlers/user/__init__.py

```diff
@@ -87,8 +96,11 @@ def user_serialize_user(session, user, language):
         'can_edit_general_settings': user.can_edit_general_settings,
         'clicked_recovery_key': user.clicked_recovery_key,
         'accepted_privacy_policy': user.accepted_privacy_policy,
-        'contexts': contexts
-
+        'contexts': contexts,
+        'fiscal_code': user.fiscal_code,
+        'external_organization_activation': ConfigFactory(session, 1).get_val('external_organization_activation'),
+        'max_msg_external_to_whistle': ConfigFactory(session, 1).get_val('max_msg_external_to_whistle'),
+        'max_msg_external_to_whistle_not_aff': ConfigFactory(session, 1).get_val('max_msg_external_to_whistle_not_aff')
```

Comment:
PERFORMANCE CONCERN

This coded should be moved out from the user serialization and put in the "public" API as it serialize some variables that pertain a global configuration.

Please not in fact that this particular code is optimized to limit the queries (here data is 

Doing 3 queries here (to fetch three variables from the Config table) becomes significantly problematic.
Regardless in fact the code could be organized to perform just one query, the problem is that this function is called in other routines to return the configurations of multiple users.

This incorrect design pattern is known to lead to problems of high algorithm complexity.